tests/ApiCallableTest.php

```diff
@@ -175,9 +175,17 @@ public function testRetryTimeoutExceeds()
             $backoffSettings);
         $callSettings = new CallSettings(['retrySettings' => $retrySettings]);
 
+        // Use time function that simulates 1100ms elapsing with each call to the stub
+        $incrementMillis = 1100;
+        $timeFuncMillis = function() use ($stub, $incrementMillis) {
```

Comment:
I haven't tried it but instead of passing a function, you could consider overriding the system timer:
http://de.php.net/manual/en/function.override-function.php